next/src/actions/admin/user-edit-general.ts

```diff
@@ -54,7 +54,7 @@ export async function userEditGeneral(
   > = {
     username: {
       value: formData.get('username'),
-      regex: /^[a-zA-Z0-9]{3,30}$/,
+      regex: /^.{2,50}$/,
```

Comment:
Poor change IMO. At least I won't see any benefits from this and previous restrictions were made for a reason.

This change:
- Does not solve any existing problem
- Adds more room to abuse usernames and usernames that will break the UI

Full name is bueno 👌